File: LandlordTenant/Tenant.m
//
//  Tenant.m
//  LandlordTenant
//
//  Created by Tye Blackie on 2017-08-21.
//  Copyright © 2017 Tye Blackie. All rights reserved.
//

#import "Tenant.h"

@implementation Tenant

-(instancetype) initWithName:(NSString*)name phoneNumber:(NSString*)phoneNumber unitNumber:(NSString*)unitNumber{
    self = [super init];
    if (self) {
        _name = name;
        _phoneNumber = phoneNumber;
        _unitNumber = unitNumber;
    }
    return self;
}

@end

File: LandlordTenant/Tenant.swift
//
//  Tenant.swift
//  LandlordTenant
//
//  Created by Tye Blackie on 2017-08-24.
//  Copyright © 2017 Tye Blackie. All rights reserved.
//

import Foundation

class Tenant{
    var name: String
    var phoneNumber: String
    var unitNumber: String
    
    init(with name: String, phoneNumber: String, and unitNumber: String) {
        self.name = name
        self.phoneNumber = phoneNumber
        self.unitNumber = unitNumber
    }
    
    
}

